frontend/asfui/app/views.py

```diff
@@ -174,6 +180,91 @@ def target_delete():
     html_template = loader.get_template( 'vd-targets.html' )
     return HttpResponse(html_template.render(context, request))
 
+
+
+
+
+@login_required(login_url="/login/")
+def alertspage(request):
+    context = {}
+    context['segment'] = 'alerts'
+    global PARSER_DEBUG
+    PARSER_DEBUG=True
+    debug(request.POST)
+    m_query = {"ignored":False}
+    mongo_username = urllib.parse.quote_plus(settings.MONGO_USER)
+    mongo_password = urllib.parse.quote_plus(settings.MONGO_PASSWORD)
+    mongo_url = settings.MONGO_URL
```

Comment:
Same comment as suggested in https://github.com/vmware-labs/attack-surface-framework/pull/18/files#diff-ccb2b112e818f612867f84573df1c02da2d80a2832dcda026b492da103e85078R45